terraform/alerting/alerts.tf

```diff
@@ -253,3 +253,85 @@ resource "google_monitoring_alert_policy" "CloudSchedulerJobFailed" {
     null_resource.manual-step-to-enable-workspace,
   ]
 }
+
+resource "google_monitoring_alert_policy" "HumanAccessedSecret" {
+  count = var.alert_on_human_accessed_secret ? 1 : 0
+
+  project      = var.project
+  display_name = "HumanAccessedSecret"
+  combiner     = "OR"
+
+  conditions {
+    display_name = "A non-service account accessed a secret."
+
+    condition_monitoring_query_language {
+      duration = "60s"
+
+      query = <<-EOT
+      fetch audited_resource
+      | metric 'logging.googleapis.com/user/human_accessed_secret'
+      | align rate(5m)
+      | every 1m
+      | group_by [resource.project_id],
+          [value_human_accessed_secret_aggregate: aggregate(value.human_accessed_secret)]
+      | condition value_human_accessed_secret_aggregate > 0
+      EOT
+
+      trigger {
+        count = 1
+      }
+    }
+  }
+
+  documentation {
+    content   = "${local.playbook_prefix}/HumanAccessedSecret.md"
+    mime_type = "text/markdown"
+  }
+
+  notification_channels = [for x in values(google_monitoring_notification_channel.channels) : x.id]
+
+  depends_on = [
+    null_resource.manual-step-to-enable-workspace,
```

Comment:
Add a dependency on the custom log based metric.

Reply:
Done